wiki/Medals/en.md

```diff
@@ -1,59 +1,295 @@
----
-needs_cleanup: true
-outdated: true
----
-<!-- TODO: new types of medals have been added but aren't listed here yet -->
-
 # Medals
 
-Medals, previously known as *achievements*, are badges on a user's profile which indicate that the user has accomplished an exceptional goal. Currently, they are divided into 7 groups: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), Beatmap Spotlights, and Seasonal Spotlights.
+Medals, previously known as *achievements*, are badges on a user's profile which indicate certain goals that have been achieved by the user. Currently, there are **247** medals available to be unlocked across 8 different categories in total: [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), [Beatmap Spotlights](#beatmap-spotlights), [Seasonal Spotlights](#seasonal-spotlights), and [Beatmap Challenge Packs](#beatmap-challenge-packs).
 
 ## Beatmap Packs
 
-The **Beatmap Packs** medals can be obtained by **playing every beatmap all the way through at least once in any difficulty** within one [Theme Beatmap Pack](https://osu.ppy.sh/beatmaps/packs?type=theme). You must play in [osu!standard](/wiki/osu!standard) to qualify a **`CLEARED`** status at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=theme) and the badge.
+The **Beatmap Packs medals** are medals that are being awarded upon the completion of certain [Theme Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=theme) or [Artist/Album Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=artist). They can be obtained by clearing every beatmap all the way through at least once in any difficulty within the corresponding beatmap pack.
 
-### Requirements
+All beatmaps in the pack have to be cleared in the [osu!](/wiki/Game_mode/osu!) game mode in order to qualify for the badge unless otherwise specified. The usage of either [difficulty reduction mods](/wiki/Game_modifier#difficulty-reduction) or [difficulty increase mods](/wiki/Game_modifier#difficulty-increase) is permitted.
 
-- The player must finish every beatmap once in any difficulty of the pack downloaded from [Theme Beatmap Pack](https://osu.ppy.sh/beatmaps/packs?type=theme).
-- The played game mode must be osu!standard.
-- Extract the theme pack from the `.zip` file and place the contents into the `Songs` folder.
-  - From here, open the osu! client and let the client parse the files before playing the beatmaps.
-  - Alternatively, the player can download the songs inside manually one by one and it would still count.
-- Any variations (like different creator) of the song is not counted.
-  It **must** be that specific song and that specific creator at that specific beatmap.
-- The player's account in question must be signed in to the osu! client (not the website).
+### List of obtainable Beatmap Packs medals
```

Comment:
```suggestion
### Obtainable Beatmap Packs medals
```
consider changing to just "Obtainable Beatmap Packs medals"; this goes for all other headers like this